app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java

```diff
@@ -58,6 +58,10 @@ public void onFinish() {
     }
 
     private static void updateTimeNotification(String machine, Context context, long timeLeft) {
+        if (notifcationIds == null || self == null) {
```

Comment:
Lol. I wonder what it is. Any theories? And can we add logs to help figure it out if it keeps happening ?